sympy/integrals/rubi/constraint.py

```diff
@@ -0,0 +1,79 @@
+from sympy.external import import_module
+matchpy = import_module("matchpy")
+
+if matchpy:
+    Constraint, substitute = matchpy.Constraint, matchpy.substitute
+else:
+    raise ImportError('MatchPy could not be imported')
```

Comment:
I have used `disabled = True` to stop running the test but it doesn't seem to be working in python2.7:

https://github.com/sympy/sympy/blob/9670ff89dd8bff7878c6d636ec4f493b87d092df/sympy/external/tests/test_sage.py#L27